app/models/action/edition_diff_factory.rb

```diff
@@ -0,0 +1,45 @@
+class Action::EditionDiffFactory
+  attr_reader :previous_draft_edition, :updated_draft_edition
```

Comment:
I don't think we need to refer to these as "draft" - I also wonder if theses need to be public?